### ISSUE TITLE
Android Play Store warning "Unoptimized APK"

### ISSUE THREAD

**Issue by Tbaut** [paritytech/parity-signer#289]
> Unoptimized APK
Warning:

>This APK results in unused code and resources being sent to users. Your app could be smaller if you used the Android App Bundle. By not optimizing your app for device configurations, your app is larger to download and install on users' devices than it needs to be. Larger apps see lower install success rates and take up storage on users' devices.

>Resolution:

>Use the [Android App Bundle](https://developer.android.com/platform/technology/app-bundle) to automatically optimize for device configurations, or manage it yourself with multiple APKs.

**Comment by geastwood** [comment]
The `aab` format is preferred by Android I guess, IIRC uploading the aab will also work on Play console, as long as you let them host the release key (which IMO it's better).

**Comment by geastwood** [comment]
Btw, having fastlane to manage these could make it really streamlined, for both iOS and Android, is there any plan integrating it?

**Comment by Tbaut** [comment]
Good point, our CI is far from being good enough right now, we'll get there, it simply hasn't been a priority.

**Comment by geastwood** [comment]
Yeh, understand. if there is a specification, I'm more than happy to jump on.
